### PR TITLE
holesky: adding MEV payloard builder and updatign node layout

### PR DESCRIPTION
**Pull request by apentori**
Adding MEV Relay for Holesky nodes.

Updating Layout based on:
* Branch (`stable`,`unstable`)
* Validator client or not
* EL type

| Host number | branch | VC | Builder | EL |
| ---- | ---- | ---- | ---- | ---- |
| geth-10 | Stable | on | On | Geth |
| Geth-09 | Stable | off | On | Geth |
| Geth-08 | unstable | off | On | Geth |
| Geth-07 | Unstable | off | On | Geth |
| Geth-10 | Unstable | off | off | Geth |
| Geth-09 | Unstable | On | off | Geth |
| Geth-08 | Stable | off | off | Geth |
| Geth-07 | Stable | On | off | Geth |
| Erigon-10 | Unstable | off | on | Erigon |
| Erigon-09 | Unstable | On | on | Erigon |
| Erigon-08 | Stable | On | on | Erigon |
| Erigon-07 | Stable | off | on | Erigon |
| Erigon-10 | Stable | off | off | Erigon |
| Erigon-09 | Stable | On | off | Erigon |
| Erigon-08 | unstable | On | off | Erigon |
| Erigon-07 | unstable | off | off | Erigon |
| Neth-10 | stable | off | on | Neth |
| Neth-09 | unstable | on | on | Neth |
| Neth-08 | unstable | on | on | Neth |
| Neth-07 | stable | on | on | Neth |
| Neth-10 | unstable | on | off | Neth |
| Neth-09 | stable | off | off | Neth |
| Neth-08 | stable | on | off | Neth |
| Neth-07 | unstable | off | off | Neth |

### CODE REVIEW

**Review by tersec — APPROVED** [review]

**Review by yakimant — APPROVED** [review]
Looks good, but please fix the lines

File: ansible/vars/layout/holesky.yml [hunk 1]
```diff
@@ -3,183 +3,183 @@
...
-    - { branch: 'testing',  start: 51000, end: 56000, el: 'erigon',     vc: true  }
-    - { branch: 'unstable', start: 56000, end: 61000, el: 'erigon',     vc: false }
-    - { branch: 'libp2p',   start: 61000, end: 66000, el: 'erigon',     vc: false }
+    - { branch: 'stable',   start: 46000, end: 51000,  el: 'erigon',     vc: false }
+    - { branch: 'testing',  start: 51000, end: 56000,  el: 'erigon',     vc: false}
+    - { branch: 'unstable', start: 56000, end: 61000,  el: 'erigon',     vc: true  , payload_builder: true }
+    - { branch: 'libp2p',   start: 61000, end: 66000,  el: 'erigon',     vc: false }
 
-  # Nethermind ---------------------------------------------------------------------
+  # Nethermind -------------------------------------- -------------------------------
```
Comment (yakimant) [inline ansible/vars/layout/holesky.yml:126]: Also

File: ansible/vars/layout/holesky.yml [hunk 2]
```diff
@@ -3,183 +3,183 @@
...
-    - { branch: 'testing',  start: 18000, end: 23000, el: 'geth',      vc: false }
-    - { branch: 'unstable', start: 23000, end: 28000, el: 'geth',      vc: false }
-    - { branch: 'libp2p',   start: 28000, end: 33000, el: 'geth',      vc: false }
+    - { branch: 'stable',   start: 13000, end: 18000,  el: 'geth',      vc: true  , payload_builder: true }
+    - { branch: 'testing',  start: 18000, end: 23000,  el: 'geth',      vc: false }
+    - { branch: 'unstable', start: 23000, end: 28000,  el: 'geth',      vc: false }
+    - { branch: 'libp2p',   start: 28000, end: 33000,  el: 'geth',      vc: false }
 
-  # Erigon ------------------------------------------------------------------------
+  # Erigon ------------------------------------------ ------------------------------
```
[the remark this answers is not in the document]
Comment (apentori) [reply ansible/vars/layout/holesky.yml]: fixed

File: ansible/vars/layout/holesky.yml (hunk 1 above)
Comment (apentori) [reply ansible/vars/layout/holesky.yml]: fixed

**Review by jakubgs — APPROVED** [review]

File: ansible/group_vars/nimbus.holesky.yml [hunk 3]
```diff
@@ -93,6 +93,10 @@ beacon_node_exec_layer_urls: ['http://localhost:{{ exec_layer_authrpc_port }}']
 beacon_node_exec_layer_jwt_secret: '{{ geth_authrpc_jwtsecret | mandatory }}'
 # Suggests it to the Execution Layer client.
 beacon_node_suggested_fee_recipient: '{{lookup("bitwarden", "nimbus/wallet/testnets", field="address")}}'
+# MEV Payload Builder
+beacon_node_payload_builder_enabled: '{{ node.get("payload_builder", false) }}'
+beacon_node_payload_builder_url: '{{ holesky_mev_boost_url }}'
+holesky_mev_boost_url: "https://boost-relay-holesky.flashbots.net/"
```
[the remark this answers is not in the document]
Comment (apentori) [reply ansible/group_vars/nimbus.holesky.yml]: No particular reason as long as we have only one url.
Varaible removed.

**Pull request merged**